meinberlin/assets/scss/components/_button.scss

```diff
@@ -89,6 +89,11 @@
     margin-bottom: $spacer;
 }
 
+.button--attached-top {
```

Comment:
from the button's point of view, it is attached to the top. The container it is in is turned by 90deg.